root/io/filemerger/CMakeLists.txt

```diff
@@ -213,7 +213,7 @@ if(${compression_default} STREQUAL "zlib")
       if(${LZ4_VERSION} VERSION_LESS "1.7.5")
             ROOTTEST_ADD_TEST(simplek-lz4-compr-level4
                            PRECMD ${ROOT_hadd_CMD} -fk404 hsimpleK404.root hsimpleK209.root hsimpleK.root hsimple.root hsimple9.root hsimple209.root hsimple409.root hsimple92.root
-                           COMMAND ${ROOT_root_CMD} -q -l -b "${CMAKE_CURRENT_SOURCE_DIR}/testSimpleFile.C(\"hsimpleK404.root\",24*25000,404,12047492,30)"
+                           COMMAND ${ROOT_root_CMD} -q -l -b "${CMAKE_CURRENT_SOURCE_DIR}/testSimpleFile.C(\"hsimpleK404.root\",24*25000,404,10042617,30)"
```

Comment:
This change is surprising large (looks like a 20% decrease).  One should check why that is the case.  (Same for two other change on line 195 and 186)